lib/sdf-server/src/service/v2/admin/migrate_connections.rs

```diff
@@ -0,0 +1,48 @@
+use axum::extract::Path;
+use dal::{
+    ChangeSetId,
+    WorkspacePk,
+    slow_rt,
+};
+use sdf_extract::PosthogEventTracker;
+use si_db::Tenancy;
+
+use crate::{
+    extract::PosthogClient,
+    service::v2::{
+        admin::{
+            AdminAPIError,
+            AdminAPIResult,
+            AdminUserContext,
+        },
+        change_set,
+    },
+    track_no_ctx,
+};
+
+pub async fn migrate_connections(
+    AdminUserContext(mut ctx): AdminUserContext,
```

Comment:
Sad we don't have an AdminChangeSetContext, but we don't have many of these endpoints yet anyway.